dts/msm8226/msm8228-samsung-r03.dts

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+        qcom,msm-id = <221 0>;
+	qcom,board-id = <0xdf08ff01 3>;
```

Comment:
https://github.com/MSM8226-Samsung/android_kernel_samsung_kmini3g/blob/cm-13.0/arch/arm/boot/dts/msm8226/msm8228-sec-atlantic3g-r03.dts#L35-L36

I guess older SoC doesn't seem to support `qcom,board-id`